sql/flow.go

```diff
@@ -23,7 +23,7 @@ import (
 
 const (
 	astroSQLCLIProjectURL     = "https://pypi.org/pypi/astro-sql-cli/json"
-	astroSQLCLIConfigURL      = "https://raw.githubusercontent.com/astronomer/astro-sdk/astro-cli/sql-cli/config/astro-cli.json"
+	astroSQLCLIConfigURL      = "https://raw.githubusercontent.com/astronomer/astro-sdk/1673-minor-version-match/sql-cli/config/astro-cli.json"
```

Comment:
This is temporary and needs to be reverted after [PR](https://github.com/astronomer/astro-sdk/pull/1706) is merged